msteams-platform/tabs/tabs-link-unfurling.md

```diff
@@ -0,0 +1,137 @@
+
+##  Stage View vs. Task module
+
+|Stage View|Task module|
+|:-----------|:-----------|
+|Stage View is useful when you have a rich content to display to the users, such as a page, a dashboard, a file, and so on. It provides  maximum real estate that helps to render your content in the full-screen canvas.|[Task module](../task-modules-and-cards/task-modules/task-modules-tabs.md) is especially useful to display messages that requires user's attention or collect information required to move to the next step.|
+  
+## Invoke the Stage View
+
+You can invoke the Stage View in the following  ways: 
+
+* Invoke Stage View from an Adaptive Card.
+* Invoke Stage View through a deep link.
+
+## Invoke Stage View from an Adaptive Card
+
+When the user enters an URL, the bot is invoked and returns an [Adaptive Card](../task-modules-and-cards/cards/cards-actions.md) with the option to open the URL in a stage. After a stage is launched, and the `tabInfo` is passed in, you can add the ability to pin the stage as a tab.  
+
+The following images display a stage opened from an Adaptive Card:
+
+![Open a stage from Adaptive Card](~/assets/images/tab-images/open-stage-from-adaptive-card1.png)  
+
+![Open a stage](~/assets/images/tab-images/open-stage-from-adaptive-card2.png)
+
+### Example 
+
+Following is the code to open a stage from an  Adaptive Card:
+
+```json
+{
+    type: "Action.Submit",
+    title: "View",
+    data: {
+          msteams: {
+            type: "invoke",
+            value: {
+                type: "tab/tabInfoAction",
+                tabInfo: {
+                    contentUrl: contentUrl,
+                    websiteUrl: websiteUrl,
+                    name: "Tasks",
+                    entityId: "entityId"
+                 }
+                }
+            }
+        }
+} 
+```
+
+The `invoke` request type must be `composeExtension/queryLink`. 
+
+> [!NOTE]
+> * `invoke` workflow is similar to the current `appLinking` workflow. 
+> * To keep consistency, it is recommended to name the `Action.Submit` as `View`.
+
+
+**To invoke Stage View**
+
+1. When the user selects **View**, the bot receives an `invoke` request. The request type is `composeExtension/queryLink`.
+1. `invoke` response from bot contains an Adaptive Card with type `tab/tabInfoAction` in it.
+1. The bot responds with a `200` code.
+
+## Invoke Stage View through deep link
+
+To invoke the deep link from your tab, you must wrap the deep link URL in `microsoftTeams.executeDeeplink(url)` API. 
+
+The following image displays a Stage View invoked through a deep link:
+
+![invoke a Stage View through a deep link](~/assets/images/tab-images/invoke-stage-view-through-deep-link.png)
+
+### Syntax 
+
+Following is the syntax of a deeplink:  
+
+https://teams.microsoft.com/l/stage/{appId}/0?context={“contentUrl”:”[contentUrl]”,“websiteUrl”:”[websiteUrl]”, “title”:”[title]”}
+
+### Examples
+
+Following are the deep link examples to invoke the Stage View:
+
+#### Example 1
```

Comment:
Examples can be in bold instead of H4. Microsoft Style doesn't distinguish between heading levels.